### PR TITLE
Add FXIOS-5351 [v109] Swiftlint: vertical parameter alignment

### PR DESCRIPTION
**Pull request by adudenamedruby**
[JIRA Reference](https://mozilla-hub.atlassian.net/browse/FXIOS-5351)

### CODE REVIEW

**Review by nishant2718 — APPROVED** [review]
Didn't know there were so many misaligned items 😱

File: Client/Frontend/Widgets/ChevronView.swift [hunk 1]
```diff
@@ -85,25 +85,29 @@ class ChevronView: UIView {
     }
 
     fileprivate func drawUpChevronAt(_ origin: CGPoint, strokeLength: CGFloat) -> UIBezierPath {
-        return drawChevron(CGPoint(x: origin.x-strokeLength, y: origin.y+strokeLength),
+        return drawChevron(
+            CGPoint(x: origin.x-strokeLength, y: origin.y+strokeLength),
```
Comment (thatswinnie) [inline Client/Frontend/Widgets/ChevronView.swift:89]: Should there be spaces before and after -/+?

Comment (adudenamedruby) [reply Client/Frontend/Widgets/ChevronView.swift:89]: good point!

**Comment by adudenamedruby** [comment]
Build was green and changes won't affect it, so saving credits:
Screenshot 2022-12-07 at 9 56 03 AM

**Pull request merged**